src/devices/networkWrappers/multipleanalogsensorsserver/MultipleAnalogSensorsServer.cpp

```diff
@@ -25,14 +25,16 @@ enum MAS_SensorTypeServer
 {
     ThreeAxisGyroscopes=0,
     ThreeAxisLinearAccelerometers=1,
-    ThreeAxisMagnetometers=2,
-    OrientationSensors=3,
-    TemperatureSensors=4,
-    SixAxisForceTorqueSensors=5,
-    ContactLoadCellArrays=6,
-    EncoderArrays=7,
-    SkinPatches=8,
-    PositionSensors=9
+    ThreeAxisAngularAccelerometers=2,
+    ThreeAxisMagnetometers=3,
+    OrientationSensors=4,
+    TemperatureSensors=5,
+    SixAxisForceTorqueSensors=6,
+    ContactLoadCellArrays=7,
+    EncoderArrays=8,
+    SkinPatches=9,
+    PositionSensors=10,
+    LinearVelocitySensors=11
```

Comment:
Can we just add the new value at the end for avoiding changing the value of the constants unless strictly necessary?

Reply:
Fixed in [8ec2a36](https://github.com/robotology/yarp/pull/3149/commits/8ec2a36b969862e466e87ce3044598b332820df3)